functions-and-operators/json-functions.md

```diff
@@ -13,28 +13,29 @@ TiDB 支持 MySQL 5.7 GA 版本发布的大多数 JSON 函数。MySQL 5.7 发布
 
 ## 创建 JSON 值的函数
 
-| 函数及语法糖                                                       | 功能描述                                                   |
+| 函数                                                              | 功能描述                                                   |
 | ------------------------------------------------------------------ | ---------------------------------------------------------- |
 | [JSON_ARRAY([val[, val] ...])][json_array]                         | 根据一系列元素创建一个 JSON 文档 |
 | [JSON_OBJECT(key, val[, key, val] ...)][json_object]               | 根据一系列 K/V 对创建一个 JSON 文档 |
 | [JSON_QUOTE(string)][json_quote]                                   | 返回一个字符串，该字符串为带引号的 JSON 值 |
 
 ## 搜索 JSON 值的函数
 
-| 函数及语法糖                                                 | 功能描述                                                     |
+| 函数                                                        | 功能描述                                                     |
 | ------------------------------------------------------------ | ------------------------------------------------------------ |
 | [JSON_CONTAINS(target, candidate[, path])][json_contains]    | 通过返回 1 或 0 来表示目标 JSON 文档中是否包含给定的 candidate JSON 文档 |
 | [JSON_CONTAINS_PATH(json_doc, one_or_all, path[, path] ...)][json_contains_path] | 通过返回 0 或 1 来表示一个 JSON 文档在给定路径是否包含数据   |
 | [JSON_EXTRACT(json_doc, path[, path] ...)][json_extract]     | 从 JSON 文档中解出某一路径对应的子文档                       |
-| [->][json_short_extract]                                     | 返回执行路径后面的 JSON 列的值；`JSON_EXTRACT(doc, path_literal)` 的语法糖 |
-| [->>][json_short_extract_unquote]                            | 返回执行路径后面的 JSON 列的值和转义后的结果； `JSON_UNQUOTE(JSON_EXTRACT(doc, path_literal))` 的语法糖 |
+| [->][json_short_extract]                                     | 返回执行路径后面的 JSON 列的值；`JSON_EXTRACT(doc, path_literal)` 的别名 |
+| [->>][json_short_extract_unquote]                            | 返回执行路径后面的 JSON 列的值和转义后的结果； `JSON_UNQUOTE(JSON_EXTRACT(doc, path_literal))` 的别名 |
 | [JSON_KEYS(json_doc[, path])][json_keys]                     | 返回从 JSON 对象的顶级值作为 JSON array 的键，如果给定了路径参数，则从选定路径中获取顶级键 |
 | [JSON_SEARCH(json_doc, one_or_all, search_str[, escape_char[, path] ...])][json_search] | 返回指定字符在 JSON 文档中的路径                             |
 
 ## 修改 JSON 值的函数
 
-| 函数及语法糖 | 功能描述 |
+| 函数        | 功能描述 |
 | --------------------------------- | ----------- |
```

Comment:
```suggestion
| --------------------------------- | ----------- |
| [JSON_APPEND(json_doc, path, value)][json_append] | `JSON_ARRAY_APPEND` 的别名 |
| [JSON_ARRAY_APPEND(json_doc, path, value)][json_array_append] | 将值追加到指定路径的 JSON 数组的末尾 |
```
Align two missing rows from [the latest English version of the document](https://docs.pingcap.com/tidb/dev/json-functions)